man/systemd.network.xml

```diff
@@ -2683,6 +2683,19 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             receiving port. When unset, the kernel's default will be used.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Isolated=</varname></term>
+          <listitem>
+            <para>Takes a boolean. Configures whether this port is
+            isolated or not. Within a bridge, isolated ports can only
+            communicate with non-isolated ports.
+            When set to true, this port cannot communicate with other
+            ports whose Isolated setting is false.
+            When set to false, this port can communicate with any other
+            ports.
+            When unset, the kernel's default will be used.</para>
```

Comment:
Seems like systemd's line length limits are longer than what you have adhered to here.